### DISCUSSION TITLE
Hitsounds persist while previewing songs in beatmap listing if playing a replay

### DISCUSSION THREAD

**Discussion by Jiekudan** [ppy/osu#30909] [Ideas]
This is probably too esoteric and minor to really matter, but I just wanted to draw some attention to it. While watching a replay, if one opens up the beatmap listing (ctrl+b) and plays the preview of a beatmap, the audio from the replay stops as expected, but the hitsounds persist. The hitsounds clash with the previewed song.

**Comment by asrieldashie** [comment]
Gotta kick this up
Been wanting to post this issue but keep forgetting since I barely bump into this during most of the playtime
